### DISCUSSION TITLE
your new u2f is goog to ka .

### DISCUSSION THREAD

**Discussion by sorrowfullys** [ravinrabbid/DonCon2040#7] [General]
Use this u2f,
It is easy to hit ka，thanks 
![image](https://github.com/user-attachments/assets/5824adf3-88f5-4cc6-8d05-97a3405684bc)

https://github.com/user-attachments/assets/020e0825-6620-48e9-8adf-99ab5ee7df70
So diy taiko is perfect 
Thanks for your doncon

**Comment by sorrowfullys** [comment]
Do you have the donconpad shell model，
i have 3d print ，and i want to print it

**Reply by ravinrabbid** [reply]
Glad to hear that it works well!

You can find the 3d print files here: https://github.com/ravinrabbid/DonCon2040/tree/main/pcb/DonConPad/Case

**Reply by sorrowfullys** [reply]
thank you very much
